PWGGA/GammaConv/AliAnalysisTaskNeutralMesonToPiPlPiMiNeutralMeson.cxx

```diff
@@ -5495,7 +5607,38 @@ void AliAnalysisTaskNeutralMesonToPiPlPiMiNeutralMeson::ProcessTrueMesonCandidat
       AliAODConversionMother mesontmp(&NDMtmp,TrueVirtualParticleCandidate);
 
       fHistoTrueMotherPiPlPiMiNDMInvMassPtFixedPzNDM[fiCut]->Fill(mesontmp.M(),mesontmp.Pt(),weighted);
-      
+
+      //Dalitz plot
```

Comment:
Also here, put the whole Dalitz plot part inside of an if(fDoMesonQA>0) condition.